pages/etat.js

```diff
@@ -0,0 +1,116 @@
+
+        <LI>Was ist der Stand der Arbeit?</LI>
+
+        <LI>Ist die Geschichte in irgendeiner Form bekannt? Andernorts erschienen? Wenn ja, warum soll sie nochmal oder nochmal anders erzählt werden?</LI>
+
+        <LI>Haben Sie die Geschichte anderen Medien angeboten?</LI>
+
+        <LI>Wer sind Sie? Wer sind Ihre Mitstreiter? Haben Sie Interessenbindungen, die Sie offenlegen sollten?</LI>
+
+        <LI>Was ist Ihr gelungenstes Projekt, und warum?</LI>
+
+      </Editorial.OL>
+
+      <P>Mailen Sie uns Ihr Exposé mit den Antworten auf all diese Fragen – so kurz wie möglich – zusammen mit Ihren Kontaktangaben an <A href='mailto:recherchen@republik.ch'>recherchen@republik.ch</A>.</P>
+
+      <Editorial.Subhead>Einige Spielregeln</Editorial.Subhead>
+
+      <P>Bitte verstehen Sie, dass wir das Geld aus dem Fonds für grosse Recherchen, Geschichten, Ideen möglichst effizient einsetzen wollen. Darum gibt es gestaffelte Budgets. Das Budget für die erste Etappe beträgt bis zu CHF 5000. Danach entscheiden wir gemeinsam, wie es weitergeht. Ist die Story fertig und geht sie in die Produktion? Ist weiterer Recherche-Aufwand nötig? Oder macht man sie mit weit mehr Ressourcen richtig gross? Was können wir von Republik an Knowhow und Technik beisteuern?</P>
+
+      <P>Die Republik verwertet die Story auf all ihren Medienkanälen vier Wochen lang exklusiv. Danach gehen die Nutzungsrechte zurück an die Urheber. Sie dürfen dann gern weiter recherchieren, schreiben, verwerten. Egal, wo die Geschichte eines Tages erscheint, sie muss das Label tragen: «Finanziert aus dem Fonds für grosse Recherchen, Geschichten und Ideen von Project R».</P>
+
+      <P>Wenn Sie am Anfang einer Geschichte stehen und nicht weiter wissen; wenn Sie Berufsanfänger sind, aber einen grossen Stoff am Wickel haben – wir beraten Sie gern. Bitte schicken Sie Ihre Fragen an recherchen@republik.ch. Sie können dort auch ein Telefonat vereinbaren mit einer unserer Redakteurinnen. Wir versprechen: Wir behandeln Ihre Informationen vertraulich. Sie können sich danach immer noch entscheiden, ob Sie die Story bei uns vorstellen möchten.</P>
+
+      <P>Der Fonds für grosse Recherchen, Geschichten und Ideen kommt von der Project R Genossenschaft, dem gemeinnützigen Dach über der Republik. Project R will die Demokratie stärken, indem sie den Journalismus als vierte Gewalt weiterentwickelt. Dem Gemeinwohl verpflichtet, fördert Project R den konstruktiven Diskurs und diskriminierungsfreie, vielfältige Debatten über die Fragen unserer Zeit.</P>
+
+      <P>Entschieden wird über die Projekte ausschliesslich innerhalb der Republik-Redaktion und ausschliesslich nach journalistischen Kriterien. Wir freuen uns, von Ihnen zu hören!</P>
+
+      <P>Das Team von Republik und <A href='https://project-r.construction'>Project R</A></P>
+
+      <P>PS: Unser <A href='https://www.republik.ch/2018/01/19/usa-serie'>erster Serienprototyp</A> wurde bereits mit Mitteln aus dem Etat finanziert.</P>
```

Comment:
could be just a relative URL:
`<A href='/2018/01/19/usa-serie'>erster Serienprototyp</A>`